invenio_rdm_records/records/jsonschemas/records/record-v6.0.0.json

```diff
@@ -341,6 +341,22 @@
               }
             }
           }
+        },
+        "_internal_notes": {
```

Comment:
because of https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/records/mappings/v7/rdmrecords/records/record-v6.0.0.json#L654
it will be still discussed, that's why it is a draft PR, too early to review since it is not the final form of the code

Reply:
thanks :)